src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,51 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION = 0;
+    private static final int VALUE = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File fromFile = new File(fromFileName);
+        int countSupply = 0;
+        int countBuy = 0;
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFile))) {
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                String[] data = value.split(",");
```

Comment:
Consider adding validation for the `data` array to ensure it has the expected length before accessing its elements. This can prevent `ArrayIndexOutOfBoundsException` if the input data is malformed.